algorithmic_efficiency/pytorch_utils.py

```diff
@@ -2,6 +2,7 @@
 from typing import Tuple
 
 from absl import logging
+import jax
 import tensorflow as tf
 import torch
 import torch.distributed as dist
```

Comment:
in pytorch_setup below, should we have N_GPUS be `max(torch.cuda.device_count(), jax.local_device_count())`? not a blocker for merging, we can fix it later if so